apps/core/templates/robots.txt

```diff
@@ -0,0 +1,6 @@
+User-Agent: *
+Disallow: /admin/
+Disallow: /search/
+Disallow: /search_json/
+
+Sitemap: {{ wagtail_site.root_url }}/sitemap.xml
```

Comment:
```suggestion
Sitemap: {{ wagtail_site.root_url }}/sitemap.xml

```

Missing a newline at the end of this file. Feels a bit overkill to me to have the robots.txt be a TemplateView so we list a sitemap using the default `sitemap.xml`, but I guess it might be helpful if we ended up with lots of pages needing multiple sitemaps.